<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffusionWindow</class>
 <widget class="QWidget" name="DiffusionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stable Diffusion</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>1260</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>521</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0;border-style:outset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
    </property>
    <property name="text">
     <string>跑图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearTextButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
    </property>
    <property name="text">
     <string>清空文本</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0;border-style:outset</string>
    </property>
    <property name="text">
     <string>请输入绘图所用文本提示：</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="GeneratedImageLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 197, 203);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UploadImageLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 197, 203);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="UploadButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>上传图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveImageButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>560</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearUploadImageButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>560</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>清空图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearImageButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>560</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>清空图片</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
